src/transformers/hf_api.py

```diff
@@ -130,13 +132,16 @@ def logout(self, token: str) -> None:
         r = requests.post(path, headers={"authorization": "Bearer {}".format(token)})
         r.raise_for_status()
 
-    def presign(self, token: str, filename: str, organization: Optional[str] = None) -> PresignedUrl:
+    def presign(self, token: str, filetype: str, filename: str, organization: Optional[str] = None) -> PresignedUrl:
         """
         HuggingFace S3-based system, used for datasets and metrics.
 
         Call HF API to get a presigned url to upload `filename` to S3.
         """
-        path = "{}/api/datasets/presign".format(self.endpoint)
+        assert filetype in self.ALLOWED_S3_FILE_TYPES, "Please specify filetype from {}".format(
+            self.ALLOWED_S3_FILE_TYPES
+        )
```

Comment:
```suggestion
        assert filetype in self.ALLOWED_S3_FILE_TYPES, f"Please specify filetype from {self.ALLOWED_S3_FILE_TYPES}"
```
Can we stop using .format and use f-strings pretty please :-)